src/Commands/Maintenance.php

```diff
@@ -75,5 +77,47 @@ public function execute(InputInterface $input, OutputInterface $output)
             Configuration::updateValue('PS_SHOP_ENABLE', 1);
             $io->success('Maintenance mode disabled');
         }
+        // list ips
```

Comment:
WDYT about doing a switch instead of multiples if ?

Reply:
Because forgetting the break is bug source.